src/lib/data/itemAliases.ts

```diff
@@ -29,9 +29,6 @@ export function setItemAlias(id: number, name: string | string[], rename = true)
 			id
 		});
 	}
-	if (Items.get(id)!.name !== firstName) {
-		throw new Error(`Failed to set item alias for item ${id}`);
-	}
```

Comment:
Removed this why?

Reply:
That wasn't part of the code when I made this branch, from what I can tell. Added it though, and fixed conflicts.